img/Icons_svg/folder-empty.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
These comments can be stripped off the code

Reply:
Sure @sunpietro. We'll compress all the svg icons